src/django/api/views/v1/production_locations.py

```diff
@@ -22,6 +23,7 @@ def __init__(self, **kwargs):
         self.opensearch_service = OpenSearchService()
         self.opensearch_query_builder = OpenSearchQueryBuilder()
 
+    @swagger_auto_schema(auto_schema=None)
```

Comment:
I recommend setting its class-level `swagger_schema` attribute to `None` to avoid duplicating the same for `def retrieve` or other methods.
Here is the documentation about it -> https://drf-yasg.readthedocs.io/en/stable/custom_spec.html#excluding-endpoints

Reply:
Done, thank you for pointing this.